Jenkinsfile

```diff
@@ -24,9 +24,21 @@ onlyOnMaster {
     }
 
     milestone()
-    stage('prod deploy') {
-        input(message: "Deploy to prod?")
-        milestone()
-        deployApp(image: img, app: 'metabase', env: 'prod', region: 'us-west-1')
+    stage("approval") {
+        input(message: "Proceed to production deploy?")
+    }
+
+    milestone()
+    stage("prod Deploy") {
+        parallel(
+            us_metabase: {
+                deployApp(image: image, app: "metabase", env: "prod", region: "us-west-1")
+            },
+            ca_metabase: {
+                // Workaround to ensure all parallel builds happen. See https://hypothes-is.slack.com/archives/CR3E3S7K8/p1625041642057400
+                sleep 2
+                deployApp(image: image, app: "metabase", env: "prod", region: "ca-central-1")
+            }
+        )
```

Comment:
Introduced a `parallel` block to deploy to `US` and `CA` regions near simultaneously. The 2 second pause was added to work around a problem where the first entry in the `parallel` block would be ignored.